src/code/views/node-value-inspector-view.coffee

```diff
@@ -46,7 +46,7 @@ module.exports = React.createClass
     SimulationStore.actions.toggledCollectorTo value
 
   updateNegativeValuesAllowed: (evt) ->
-    value = !evt.target.checked   # checkbox is reverse polarity from property
+    value = !!evt.target.checked
```

Comment:
Did you mean to use `!!``?  That just converts a truthy/falsy value to a boolean and checked is already a boolean.